assets/css/base/_forms.scss

```diff
@@ -0,0 +1,21 @@
+.form {
+  @apply block;
+
+  &__label {
+    @apply block text-gray-700 text-sm font-bold mb-2;
+
+    &--uppercase {
+      @apply uppercase;
+    }
+  }
+
+  &__input {
+    @apply block appearance-none w-full py-3 px-4 mb-3 leading-tight text-gray-700 bg-gray-50 focus:bg-white border border-gray-200 focus:border-gray-500 rounded focus:outline-none;
+  }
+}
+
+.invalid-feedback {
```

Comment:
Default error class used by phoenix when there's a changeset error.

Reply:
I remember somehow we could override this class name @bterone 
https://github.com/phoenixframework/phoenix/blob/6605625065e2ea924bb0458335674756ad67ccdf/installer/templates/phx_web/views/error_helpers.ex#L14

Reply:
It allows us to customize it in the `error_helpers.ex` but I thought it was a little weird to change it there. I'll just change it there for now since it's always going to have the same class name 👍